docs/application/dotnet/guides/personal/authentication.md

```diff
@@ -1,17 +1,13 @@
 # Authentication and Authorization
 
 
-The authentication and authorization features introduce how you can obtain access tokens using OAuth 2.0, and authenticate users using the FIDO framework.
 
 You can use the following authentication and authorization features in your .NET applications:
```

Comment:
How about removing authorization from this sentence?

```suggestion
You can use the following authentication features in your .NET applications:
```

Reply:
changes are done, kindly review